Sources/Swifter.swift

```diff
@@ -56,6 +56,27 @@ public enum TwitterURL {
     
 }
 
+// MARK: - Tweet Mode
+public enum TweetMode {
+    case defaultValue
```

Comment:
You can do the following to allow the keyword `default` to be used instead of having a longer name.
```
case `default`
```